docs/spark/tutorials/get-started-windows-instructions.md

```diff
@@ -0,0 +1,108 @@
+
+> [!div class="checklist"]
+>
+> * Pre-requisites to run your .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run your .NET for Apache Spark application
+
+## Pre-requisites
+
+- Download and install the following products: **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** | **[Visual Studio 2019](https://www.visualstudio.com/downloads/)** | **[Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)** | **[Apache Spark 2.4.1](https://archive.apache.org/dist/spark/spark-2.4.1/spark-2.4.1-bin-hadoop2.7.tgz)**
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (for example, `c:\bin\Microsoft.Spark.Worker\`).
+
+ > [!IMPORTANT]
+ >
+ > Create a [new environment variable](https://www.java.com/en/download/help/path.xml) `DOTNET_WORKER_DIR` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (for example, *c:\bin\Microsoft.Spark.Worker*).
+
+For detailed instructions, you can see [Building .NET for Apache Spark from Source on Windows](../how-to-guides/windows-instructions.md).
+
+## Authoring a .NET for Apache Spark App
+
+1. Open Visual Studio > **Create New Project** > **Console App (.NET Core)** > Name: `HelloSpark`.
+
+   For detailed instructions, you can see [How to create the app in Visual Studio](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio?tabs=csharp).
+
+1. Install `Microsoft.Spark` Nuget package into the solution from the [spark nuget.org feed](https://www.nuget.org/profiles/spark) - see [Ways to install Nuget Package](https://docs.microsoft.com/nuget/consume-packages/ways-to-install-a-package)
+1. Write the following code into *Program.cs*:
+
+    ```csharp
+    using Microsoft.Spark.Sql;
+
+    namespace HelloSpark
+    {
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                SparkSession spark = SparkSession.Builder().GetOrCreate();
+                DataFrame df = spark.Read().Json("people.json");
+                df.Show();
+            }
+        }
+    }
+    ```
+
+1. Build the solution
+
+## Running your .NET for Apache Spark App
+
+1. Open your terminal and navigate into your app folder:
+
+    ```
+    cd <your-app-output-directory>
+    ```
+    
+1. Create *people.json* with the following content:
+
+    ```json
+    {"name":"Michael"}
+    {"name":"Andy", "age":30}
+    {"name":"Justin", "age":19}
+    ```
+    
+1. Run your app
+
+    ```batch
+    spark-submit `
+    --class org.apache.spark.deploy.dotnet.DotnetRunner `
+    --master local `
+    microsoft-spark-2.4.x-<version>.jar `
+    dotnet HelloSpark.dll
+    ```
```

Comment:
I think you meant bash ? Unless there is a `batch` identifier that I'm not aware of.
I also don't understand the back tick at the end of each line. I suspect it should be a backslash to escape the new line ?

Reply:
Yes, changed it to bash and added the backslash to escape newline. Thanks!